src/main/scala/ai/eto/rikai/sql/model/torchhub/TorchHubRegistry.scala

```diff
@@ -16,11 +16,29 @@
 
 package ai.eto.rikai.sql.model.torchhub
 
-import ai.eto.rikai.sql.model.PyImplRegistry
+import ai.eto.rikai.RikaiConf
+import ai.eto.rikai.sql.model.{
+  Model,
+  ModelNotFoundException,
+  ModelSpec,
+  PyImplRegistry
+}
+import org.apache.spark.sql.SparkSession
 
 /** TorchHub-based Model [[Registry]].
   */
 class TorchHubRegistry(val conf: Map[String, String]) extends PyImplRegistry {
+  override def resolve(session: SparkSession, spec: ModelSpec): Model = {
```

Comment:
why not just remove torchhub impl from `DEFAULT_REGISTRIES` in Registry.scala? Isn't that the only thing you need to disable it by default?

Reply:
because here i am printing a friendly error message

Reply:
you suggested impl is not friendly，people will not be hinted on how to enable it